lidar_obstacle_detection/GDXBoxMessage[] boxes
